images/scripts/arch.setup

```diff
@@ -135,7 +135,9 @@ useradd -c Builder builder
 echo "builder ALL = NOPASSWD: /usr/sbin/makechrootpkg, /usr/sbin/mkarchroot, /usr/bin/*-x86_64-build" > /etc/sudoers.d/90-archbuild
 
 BUILD_DEPS="\
+accountsservice \
 base-devel \
+cockpit \
```

Comment:
AFAIUI the [invocation](https://github.com/cockpit-project/bots/blob/c940e7f713791c7c0a22a03cbdb9d93ec3947c45/images/scripts/arch.setup#L165) this is a direct package list, not a list of packages whose build depends you want to install -- the latter wouldn't make sense, we don't ever want to build glib-networking or podman. So I totally get accountsservice and possibly podman, but none of our packages ought to *build-depend* on cockpit?

Reply:
This is a bit of an issue with the arch build system which expects `depends` in the build chroot, previously this just worked as /var/cache/pacman/pkg contains a copy of `cockpit` see for example:

https://cockpit-logs.us-east-1.linodeobjects.com/pull-1903-3fef1233-20241104-190321-arch/log look for line `resolving dependencies`. So removing the package cache would break this.

An alternative solution is to pass `--nodeps` to makepkg via `makechrootpkg` that should work (but accountservice should be included as it's a required dep)

Reply:
Uh, interesting. So this is e.g. for building cockpit-podman which has `depends=(cockpit podman)`?

Reply:
Yup exactly.